src/utils.test.js

```diff
@@ -113,3 +117,52 @@ describe('getQueryParameters', () => {
     });
   });
 });
+
+describe('generateAPIConfigUrl', () => {
+  beforeEach(() => {
+    config = getConfig();
+    delete window.location;
+    window.location = new URL('https://apps.testing.example.com');
+  });
+
+  it('should not set mfe configuration', () => {
+    expect(generateAPIConfigUrl()).toEqual(null);
+  });
+
+  it('shopuld not match url with mfe config api regex', () => {
+    config.MFE_CONFIG_API_PATH = 'api/mfe/v1/config';
```

Comment:
MFE_CONFIG_API_PATH only works if doesnt start  with / and  end with /.
https://github.com/eduNEXT/frontend-platform/pull/5/files#diff-3274f1a37032fb0ae4e2823def0007c634e869ae0dfc304ff6a12c36513c3a52R203

Reply:
there is no validations for that the path shouldn't start with /